### DISCUSSION TITLE
Being able to add extension

### DISCUSSION THREAD

**Discussion by Remi-Avec-Un-I** [tw93/Pake#247] [Ideas]
Hello, 
Would it be possible to add the possibily to add extension, during the compiling for exemple? Like Mozilla FireFox extension, or Chrome etc...

**Comment by tw93** [comment]
Due to the underlying framework, this feature should not be supported.
https://github.com/tauri-apps/tauri/discussions/2685

**Reply by Remi-Avec-Un-I** [reply]
So there is no way to block trackers and advertising? And thank you for your answer, and thank you for your work

**Reply by tw93** [reply]
here https://github.com/tw93/Pake/wiki/Advanced-Usage-of-Make#1-how-do-i-rewrite-the-style-eg-to-remove-ads-from-the-original-site-or-even-redesign-it

**Reply by Remi-Avec-Un-I** [reply]
Perfect ! ty
Edit : I tryed to create npm run dev:debug but it is not creating the debug for the website that i'm looking for. I don't know where to specify it

**Reply by tw93** [reply]
<img width="1312" alt="image" src="https://user-images.githubusercontent.com/8736212/211819964-3aed7367-85eb-42fd-b127-75a947b4e078.png">
Like this

**Reply by Remi-Avec-Un-I** [reply]
No, I mean I don't want to be on WeRead, but on youtube.com website
